erpnext/templates/pages/task_info.html

```
@@ -15,66 +15,51 @@
 		<input type="hidden" name="web_form" value="tasks">
 		<input type="hidden" name="doctype" value="Task">
-		<input type="hidden" name="name" value="TASK00056">
+		<input type="hidden" name="name" value="{{ doc.name }}">
 
 		<div class="row">
 			<div class="col-sm-12" style="max-width: 500px;">
 				<div class="form-group">
-					<label for="project" class="control-label text-muted small">{{ __("Project") }}</label>
-						<input type="text" class="form-control" name="project" readonly value= "{{ doc.project }}">
+					<label for="project" class="control-label text-muted small">{{ _("Project") }}</label>
+						<input type="text" class="form-control" name="project" readonly value= "{{ doc.project or "" }}">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)